ext/json/lib/json/add/date_time.rb

```diff
@@ -63,5 +63,3 @@ def to_json(*args)
     as_json.to_json(*args)
   end
 end
-
```

Comment:
I think it would be best to avoid unnecessary whitespace changes in this pull request, if possible.

Reply:
Removed 👍